src/components/GameView/CannotCounterDialog.vue

```diff
@@ -1,31 +1,31 @@
 <template>
-  <base-dialog
+  <BaseDialog
     v-if="oneOff"
     id="cannot-counter-dialog"
     v-model="show"
     title="Cannot Counter"
   >
     <template #body>
       <div v-if="!opponentLastTwo" class="my-2">
-        Your opponent has played the
-        <game-card-name :card-name="oneOff.name" />
+        Your opponent has played the 
+        <GameCardName :card-name="oneOff.name" />
         as a one-off
-        <span v-if="target">
-          targeting your
-          <game-card-name :card-name="target.name" />
+        <span v-if="target"> 
+          targeting your 
```

Comment:
Don't think we need the whitespace
```suggestion
        <span v-if="target">
          targeting your
```